sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BatchRequestResponseConstants.java

```diff
@@ -14,7 +14,9 @@
     public static final int MAX_OPERATIONS_IN_DIRECT_MODE_BATCH_REQUEST = 100;
 
     public static final int DEFAULT_MAX_MICRO_BATCH_INTERVAL_IN_MILLISECONDS = 100;
-    public static final int DEFAULT_MAX_MICRO_BATCH_CONCURRENCY = 2;
+    public static final int DEFAULT_MAX_MICRO_BATCH_CONCURRENCY = 1;
```

Comment:
why change from 2 to 1?

Reply:
Because concurrency of 2 is pretty much always wrong. Even a single client with 4 cores can easily saturate the 10,000 RU of a physical partition.

Reply:
So this change was intentional - but leaving the comment open for others to chime in as well.

Reply:
if the concurrency should always be 1, then should we remove the setMaxMicroBatchConcurrency public api from bulkOption?

Reply:
The only scenario where I think a higher maxConcurrency makes sense is if you build a webservice accepting requests containing info that results in multiple documents you want to ingest - so where your call to processBulkOperations would only contain let's say a couple dozen/hundred documents. I can imagine that latency might be better with higher concurrency - but this is an edge case - the concurrency can still be modified by customers but 1 as default seems to be meeting most scenarios better